var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -73,6 +74,8 @@ class Vtk(CMakePackage):
 
     # VTK will need Qt5OpenGL, and qt needs '-opengl' for that
     depends_on('qt+opengl', when='+qt')
+    
```

Comment:
flake8 error on this line (trailing whitespaces)
https://travis-ci.org/spack/spack/jobs/440071587